internal/postgres/reconcile_test.go

```diff
@@ -251,6 +251,7 @@ initContainers:
     halt "$(permissions "${pgbrLog_directory}" ||:)"
     install -D --mode=0600 -t "/tmp/replication" "/pgconf/tls/replication"/{tls.crt,tls.key,ca.crt}
 
+
```

Comment:
Accidental check-in?

Reply:
No, that's actually required for the test. When TDE is not enabled, an extra newline is added due to the way the change is made. It follows a similar pattern as the existing [tablespace command](https://github.com/CrunchyData/postgres-operator/blob/master/internal/postgres/config.go#L335).

Reply:
Gotcha. Thanks!